src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/ContextInfos/PackageSourceContextInfo.cs

```diff
@@ -48,7 +48,7 @@ public bool Equals(PackageSourceContextInfo other)
                 return false;
             }
 
-            return Name.Equals(other.Name, StringComparison.CurrentCultureIgnoreCase) &&
```

Comment:
Name comparison of source.

Reply:
This one also looks irrelevant change, reverting because  it's already doing `Source` compare with `StringComparison.OrdinalIgnoreCase`. Here actually `Source` is more important than `Name`.